yarn-project/circuits.js/src/cbind/circuits.gen.ts

```diff
@@ -3208,7 +3208,7 @@ export function abisComputeGlobalsHash(wasm: IWasmModule, arg0: GlobalVariables)
 export function abisComputePublicDataTreeValue(wasm: IWasmModule, arg0: Fr): Fr {
   return Fr.fromBuffer(callCbind(wasm, 'abis__compute_public_data_tree_value', [toBuffer(arg0)]));
 }
-export function abisComputePublicDataTreeIndex(wasm: IWasmModule, arg0: Fr, arg1: Fr): Fr {
+export function abisComputePublicDataTreeIndex(wasm: IWasmModule, arg0: Address, arg1: Fr): Fr {
```

Comment:
Is this `Address` or `AztecAddress`?

Reply:
This is correct because of [this alias](https://github.com/AztecProtocol/aztec-packages/blob/4b722a5d79143fa4138c7c7fded119945b23d5f7/yarn-project/circuits.js/src/cbind/types.ts#L81). There is no AztecAddress on the C++ side so I guess Adam set it up like this to make it work.

Reply:
Got it